packages/nx-plugin/src/executors/cli/executor.unit.test.ts

```diff
@@ -67,10 +69,17 @@ describe('runAutorunExecutor', () => {
     );
     expect(output.success).toBe(true);
     expect(output.command).toMatch('utils');
-    // eslint-disable-next-line n/no-sync
-    expect(execSync).toHaveBeenCalledWith(expect.stringContaining('utils'), {
-      cwd: 'cwd-form-context',
-    });
+    expect(executeProcessSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        command: 'npx',
+        args: expect.arrayContaining(['@code-pushup/cli']),
+        cwd: 'cwd-form-context',
+        observer: expect.objectContaining({
+          onError: expect.any(Function),
+          onStdout: expect.any(Function),
+        }),
+      }),
+    );
```

Comment:
See previous comment.

```suggestion
    expect(executeProcessSpy).toHaveBeenCalledWith({
      command: 'npx',
      args: expect.arrayContaining(['@code-pushup/cli']),
      cwd: 'cwd-form-context',
      observer: {
        onError: expect.any(Function),
        onStdout: expect.any(Function),
      },
    });
```